cmake/Findutmps.cmake

```diff
@@ -0,0 +1,38 @@
+# - Try to find utmps and its dependencies
+# Once done this will define
+# UTMPS_FOUND - system has utmps and dependencies
+# UTMPS_INCLUDE_DIR - the utmps include dir
+# UTMPS_LIBRARIES - libutmps, libskarnet library and sysdeps libraries
+
+if (UTMPS_INCLUDE_DIR AND UTMPS_LIBRARIES)
+    # Already cached
+    set(UTMPS_FIND_SILENTLY TRUE)
+endif (UTMPS_INCLUDE_DIR AND UTMPS_LIBRARIES)
+
+find_path(UTMPS_INCLUDE_DIR NAMES utmps/utmpx.h)
```

Comment:
I think only

```
find_library(UTMPS_LIBRARY utmps)
find_path(UTMPS_INCLUDE_DIR NAMES utmps/utmpx.h)
```

are actually necessary. The include doesn't have any dependencies and as long as the library is linked dynamically it's enough to mention just that.

Reply:
I've changed this part too. Or should I remove UTMPS_FIND_SILENTLY too? I've looked at FindPAM.cmake while writing this so I've borrowed this part.

Reply:
IMO everything except lines 12-18 can be removed, though I'm not familiar enough with cmake whether there are some edge cases or expectations that need to be met by find modules.

Reply:
Separate find module file is fine IMO